tests/dem/particle_wall_contact_force_linear.output

```diff
@@ -1,2 +1,2 @@
 
-DEAL::The contact force acting on particle 1 is: 6.28532 N 
```

Comment:
Why is the diff way bigger than de last diff? You use the same model than better for the kn, mhm